From bfa762153966cbe92cf78449a807a38f7df2422a Mon Sep 17 00:00:00 2001
From: Alexsey Shestacov <wingrime@gmail.com>
Date: Sat, 20 Oct 2012 20:48:14 +0600
Subject: [PATCH 823/944] sunxi: nand: fix build warnings

drivers/block/sunxi_nand/nfd/nand_blk.c: In function 'init_blklayer':
drivers/block/sunxi_nand/nfd/nand_blk.c:1333: warning: unused variable 'cmu_clk'
drivers/block/sunxi_nand/nfd/nand_blk.c: At top level:
drivers/block/sunxi_nand/nfd/nand_blk.c:120: warning: 'ahb_nand_clk' defined but not used
drivers/block/sunxi_nand/nfd/nand_blk.c:121: warning: 'mod_nand_clk' defined but not used
---
 drivers/block/sunxi_nand/nfd/nand_blk.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/block/sunxi_nand/nfd/nand_blk.c b/drivers/block/sunxi_nand/nfd/nand_blk.c
index 317f301..ee915ef 100644
--- a/drivers/block/sunxi_nand/nfd/nand_blk.c
+++ b/drivers/block/sunxi_nand/nfd/nand_blk.c
@@ -117,9 +117,10 @@ struct collect_ops collect_arg;
 DEFINE_SEMAPHORE(nand_mutex);
 static unsigned char volatile IS_IDLE = 1;
 u32 nand_handle=0;
+#ifdef USE_SYS_CLK
 static struct clk *ahb_nand_clk = NULL;
 static struct clk *mod_nand_clk = NULL;
-
+#endif
 static int nand_flush(struct nand_blk_dev *dev);
 
 spinlock_t     nand_rb_lock;
@@ -1330,10 +1331,7 @@ static int __init init_blklayer(void)
 	unsigned long irqflags;
 
 	#ifndef USE_SYS_CLK
-		__u32 cmu_clk;
 		__u32 nand_clk;
-		//set nand clk
-		//cmu_clk = get_cmu_clk();
 		set_nand_clock(20);
 	#else
 		ret = nand_request_clk();
-- 
1.8.0

